packages/ubuntu_bootstrap/assets/ubuntu-provision.yml

```diff
@@ -0,0 +1,33 @@
+pages:
+  locale:
+    image: "packages/ubuntu_provision/assets/mascot.png"
+    title: "Welcome"
+    visible: true
+  welcome:
+    title: "Welcome to Ubuntu"
+    visible: true
+  rst:
+    image: "assets/storage/rst.svg"
+    title: "Rapid Storage Technology"
+    visible: true
+  keyboard:
+    title: "Keyboard layout"
+    visible: true
+  network:
+    title: "Connect to a network"
+    visible: true
+  refresh:
+    title: "Update available"
+    visible: true
+  source:
+    title: "Type of installation"
+    visible: true
+  secureBoot:
+    title: "Configure Secure Boot"
+    visible: true
+  storage:
+    title: "Ready to install"
```

Comment:
Even though the titles aren't used yet, the storage page should probably have a different one

Reply:
Fixed! Probably a copy paste mistake